libr/core/panels.c

```diff
@@ -681,7 +681,7 @@ static void panelPrint(RCore *core, RConsCanvas *can, RPanel *panel, int color)
 	}
 }
 
-static void menuPanelPrint(RConsCanvas *can, RPanel *panel, int x, int y, int w, int h) {
+void __menuPanelPrint(RConsCanvas *can, RPanel *panel, int x, int y, int w, int h) {
```

Comment:
uh? why drop static?.. actually this `__` rule seems a bit verbose in this use case . maybe we shouldnt go this way, or define a conditional rule when 3rd party apis (non-r2 ones) are also used  to make clear its a local function when reading the code.